libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -18,8 +18,10 @@ userdata Location method get_vector_from_origin_NEU boolean Vector3f'Null
 userdata Location method get_bearing float Location
 userdata Location method get_distance_NED Vector3f Location
 userdata Location method get_distance_NE Vector2f Location
+userdata Location method get_alt_frame int32_t'skip_check
 userdata Location method change_alt_frame boolean Location::AltFrame'enum Location::AltFrame::ABSOLUTE Location::AltFrame::ABOVE_TERRAIN
 userdata Location method copy Location
+userdata Location method longitude_scale float int32_t'skip_check
```

Comment:
whats the use case? Would be quite easy to duplicate this helper in lua.

Reply:
Use case is to move a defined reach north or south without changing the length.
I thought the code is still in memory and we should use it. It's easy to duplicate, but do we want to have redundancies?

Reply:
Each lua binding costs flash for everyone so we try not to add bindings for stuff that is used infrequently and can be easily replicated with lua.

Reply:
Thank you for the quick reply Pete, that surely makes sense. So I will build the function in lua.